From 0b02b186c87f3805c404193d6b57cd4af2ca3640 Mon Sep 17 00:00:00 2001
Message-Id: <patch-v6-14.14-0b02b186c87-20220525T100743Z-avarab@gmail.com>
In-Reply-To: <cover-v6-00.14-00000000000-20220525T100743Z-avarab@gmail.com>
References: <RFC-cover-v5-00.10-00000000000-20220421T183001Z-avarab@gmail.com>
	<cover-v6-00.14-00000000000-20220525T100743Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Wed, 25 May 2022 10:08:26 +0200
Subject: [PATCH v6 14/14] CI: make the --github-workflow-markup "github"
 output the default
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

As noted in the preceding commit there's outstanding issues with this
output target over the "raw" one, but let's flip the default so that
the end-state of this series matches that of [1]. If we'd like we can
now easily revert back.

My ci/config/print-test-failures-output-type on my "ci-config" branch
is currently the following script, which allows for naming branches
with _CI_OUT_{GITHUB,RAW} suffixes to force a given output type:

	#!/bin/sh

	set -ex

	ref=$1
	shift
	echo "ref: $ref" >&2
	echo "arguments: $@" >&2

	case "$ref" in
	*_CI_OUT_GITHUB)
		echo github
		;;
	*_CI_OUT_RAW)
		echo raw
		;;
	esac

1. https://lore.kernel.org/git/pull.1117.v3.git.1653171536.gitgitgadget@gmail.com/

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 .github/workflows/main.yml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.github/workflows/main.yml b/.github/workflows/main.yml
index 4195f37c1ba..39675cbdd60 100644
--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -26,7 +26,7 @@ jobs:
       - id: print-test-failures-output-type
         name: check what output type ci/print-test-failures.sh uses
         run: |
-          type_default=raw
+          type_default=github
           type=$type_default
 
           if test -x config-repo/ci/config/print-test-failures-output-type
-- 
2.36.1.1045.gf356b5617dd

